IPython/html/static/edit/js/main.js

```diff
@@ -29,22 +29,29 @@ require([
     ){
     page = new page.Page();
 
-    var base_url = utils.get_body_data('baseUrl');
-    var file_path = utils.get_body_data('filePath');
-    contents = new contents.Contents({base_url: base_url});
-    var config = new configmod.ConfigSection('edit', {base_url: base_url});
+    var common_options = {
+        ws_url : utils.get_body_data("wsUrl"),
+        base_url : utils.get_body_data("baseUrl"),
+        file_path : utils.get_body_data('filePath'),
+        notebook_path : utils.get_body_data("notebookPath"),
+        notebook_name : utils.get_body_data('notebookName')
```

Comment:
common_options is *not* an API thing. There's no need for it to be consistent. It is purely a convenient thing when we want to reuse a few keyword arguments.

This especially doesn't make sense because the editor won't have a notebook path or name.